index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Ooyala Receiver</title>
+
+    <script type="text/javascript" src="//www.gstatic.com/cast/sdk/libs/receiver/2.0.0/cast_receiver.js"></script>
+    <script type="text/javascript" src="//player.ooyala.com/core/e48c9b51282d406f957c666149763424?plugins=bm,cast"></script>
+</head>
+
+<body>
+    <div id="splash-screen" class="fbox-container fs">
+        <img id="splash-image" src="images/logo.svg" />
+        <div id="status-cast">
+        </div>
+    </div>
+    <div id="player"></div>
+    <script type="text/javascript" src="//player.ooyala.com/static/chromecast/latest/cast.min.js"></script>
+    <script>
+        var pageLevelParams = {
+            skin: {
+                inline: {
+                    pauseScreen: {
+                        showDescription: false
+                    }
+                }
+            },
+            onCreate: player =>
+                player.mb.subscribe(
+                    '*',
+                    'urn:x-cast:testnamespace',
+                    event => {
+                        console.log('Event from outer onCreate function', event);
+                    }
+                )
+        };
+
+        OOCast.init(
+            'urn:x-cast:testnamespace',
```

Comment:
this should be the same namespace used by ooyala sender apps (urn:x-cast:ooyala) to be able to work on QA side

Reply:
Sure, will do.